src/Autocomplete/Autocomplete.js

```diff
@@ -217,11 +217,15 @@ export class Autocomplete extends Component {
       case TAB:
         shouldOpenOptions = false;
         this.selectCurrentOption();
+        this.hideOptions();
         break;
       case ESC:
         shouldOpenOptions = false;
         this.hideOptions();
         break;
+      case SHIFT:
```

Comment:
Is this behavior tested?
What is this change doing?

Reply:
It is when you go back in the tabs

Reply:
No it does not have a test, I can add it, silly one

Reply:
Cool, thank you!